### PR TITLE
added Contact Us section , fixed issue #71

### LINKED ISSUE #71: Adding functionality for contactUs section
# Description
We have a `Contact us` section but it doesn't do anything yet. Adding some non-void functionality to this section will enhance the app. 

# Relevant files
lib/Screens/Contact.dart

# Screenshot
NotApplicable

# References
NotApplicable

### PR DESCRIPTION
**Pull request by Pranav108**
## Program
- [x] CodePeak
- [ ] Script Winter of Code
- [ ] None

## Issue
Fixes #71 

## Description
Added UI and features for `contactUs` section.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change

## Screenshot
drawing

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] The app runs properly
- [x] I have performed a self-review of my own code
- [x] I have formatted and commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: lib/Screens/Contact.dart [hunk 1]
```diff
@@ -1,10 +1,144 @@
...
+              ),
+            ),
+            Flexible(
+              flex: 2,
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: [
+                  ClickableCard(
+                    textContent: 'Fork our github repository',
```
Comment (Saransh-cpp) [inline lib/Screens/Contact.dart:52]:
```suggestion
                    textContent: 'Visit our github repository',
```

File: lib/Screens/Contact.dart [hunk 2]
```diff
@@ -1,10 +1,144 @@
...
+                  ClickableCard(
+                    textContent: 'Suggest some feature',
+                    color: Colors.purple[300],
+                    onTap: () async {
+                      await _launchURL(
+                          'https://github.com/Saransh-cpp/BookRentApp-Chapter3/issues');
+                    },
+                  ),
+                  ClickableCard(
+                    textContent: 'Contact owner on Github',
```
Comment (Saransh-cpp) [inline lib/Screens/Contact.dart:68]:
```suggestion
                    textContent: 'Contact author on Github',
```

I don't own all of the code written in here :)

File: lib/Screens/Contact.dart [hunk 3]
```diff
@@ -1,10 +1,144 @@
...
+                    color: Colors.blueGrey,
+                    onTap: () async {
+                      await _launchURL('https://github.com/Saransh-cpp');
+                    },
+                  ),
+                  ClickableCard(
+                    textContent: 'Contact owner on LinkedIn',
```
Comment (Saransh-cpp) [inline lib/Screens/Contact.dart:75]:
```suggestion
                    textContent: 'Contact author on LinkedIn',
```

File: lib/Screens/Contact.dart [hunk 4]
```diff
@@ -1,10 +1,144 @@
...
+    if (await canLaunch(url)) {
+      await launch(url);
+    } else {
+      ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+        content: Text(
+          'Something went wrong!',
+          style: TextStyle(fontSize: 20),
+        ),
+        backgroundColor: Colors.grey[700],
+        behavior: SnackBarBehavior.floating,
```
Comment (Saransh-cpp) [inline lib/Screens/Contact.dart:101]:
```suggestion
```

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]
Thanks, @Pranav108! Awesome work! Some minor comments below -

**Comment by Pranav108** [comment]
I also come up with a idea to refer them on play-store for providing feedback and rating (app link will be added later).
Should I add it?

**Comment by Saransh-cpp** [comment]
Yes, you can add that!

**Comment by Pranav108** [comment]
> Yes, you can add that!

I have improved the code and updated the screenshot in PR.

**Comment by Saransh-cpp** [comment]
@Pranav108. I think you replaced the contact page with the cart screen by mistake.

**Review by Saransh-cpp — APPROVED** [review]
Awesome, @Pranav108!

**Pull request merged**